.github/workflows/ci.yml

```diff
@@ -51,20 +51,3 @@ jobs:
         id: get_package_name
       - run: "echo -n ::set-output name=version::; dpkg-parsechangelog -l./debian/changelog | sed -n 's/^Version: //p'"
         id: get_version
-      - name: Upload deb package to Bintray
```

Comment:
should we also remove the get_package_name and get_version steps? they were only used for uploading to bintray

Reply:
Let's better remove the whole `build_deps`

Reply:
my only concern with that is that we won't know if we've broken building dpkgs externally (and in a way, internally since if `make itest_%` fails on GH Actions it could signal that it'll also fail in our jenkins environment) by removing building debs from our OSS testing

Reply:
Oh, right, we don't have separate `itest_*` steps anymore...